docs/userguide/async.md

```diff
@@ -0,0 +1,74 @@
+# Asynchronous Workflows
+
+Asynchronous workflows describe situations where the loop between measurements and
+recommendations is more complex and needs to incorporate various other aspects. These
+could for instance be:
+- **Distributed workflows**: When recommendations are distributed across several
+  operators, e.g. at different locations or in several reactors, some experiments might
+  have been started, but are not ready when the next recommendation is requested.
+  Without further consideration, the algorithm would be likely to recommend the pending
+  measurements again, as it is unaware they were already started.
+- **Partial targets**: Relevant when dealing with multiple targets which require very
+  different amounts of time to measure. When a new recommendation is requested, the
+  targets of previously recommended points might only partially available, but that
+  info should ideally already be considered.
+
+
+## Pending Measurements
+
+With *pending measurements* we mean recommendations whose measurement process has
+been started, but not yet completed by time of triggering the next set of
+recommendations. This is not to be confused with a situation where the targets of a
+recommendation have been partially measured. A measurement is *pending* whenever one or
+more of its targets are not measured yet. Only once all the targets are available,
+the data point can be removed from `pending_measurements` and be provided via 
+`measurements`.
+
+To avoid repeated recommendations in the above scenario, BayBE provides the 
+`pending_measurements` keyword. It is available in `.recommend` from
+[`Campaign`](baybe.campaign.Campaign) or any
+[`Recommender`](baybe.recommenders.base.RecommenderProtocol).
```

Comment:
You could actually make the links point directly to the methods

Reply:
done